15314f1ddb882700dc9fabc5ca961943/update/sys_script_4845d98c97600a10b7daf896f053afb9.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?><record_update sys_domain="global" table="sys_script">
+    <sys_script action="INSERT_OR_UPDATE">
+        <abort_action>false</abort_action>
+        <access>package_private</access>
+        <action_delete>false</action_delete>
+        <action_insert>false</action_insert>
+        <action_query>false</action_query>
+        <action_update>true</action_update>
+        <active>false</active>
+        <add_message>false</add_message>
+        <advanced>true</advanced>
+        <change_fields>false</change_fields>
+        <client_callable>false</client_callable>
+        <collection>x_illu2_illumio_illumio_scheduled_jobs</collection>
+        <condition/>
+        <description/>
+        <execute_function>false</execute_function>
+        <filter_condition table="x_illu2_illumio_illumio_scheduled_jobs">job_type=connectivity_check^job_status=completed^EQ<item display_value="Connectivity Check" endquery="false" field="job_type" goto="false" newquery="false" operator="=" or="false" value="connectivity_check"/>
+            <item display_value="Completed" endquery="false" field="job_status" goto="false" newquery="false" operator="=" or="false" value="completed"/>
+            <item endquery="true" field="" goto="false" newquery="false" operator="=" or="false" value=""/>
+        </filter_condition>
+        <is_rest>false</is_rest>
+        <message/>
+        <name>Initial fetch of label dimensions</name>
+        <order>100</order>
+        <priority>100</priority>
+        <rest_method/>
+        <rest_method_text/>
+        <rest_service/>
+        <rest_service_text/>
+        <rest_variables/>
+        <role_conditions/>
+        <script><![CDATA[(function executeRule(current, previous /*null when async*/ ) {
+    // Fetch the label dimensions from Illumio PCE for this configuration
+    try {
+		var IlUtils = new x_illu2_illumio.IllumioUtils();
```

Comment:
1. please use better name here instead of "IlUtils"
2. pls fix indentation
3. please use better name than "x_illu2_illumio_illumio_pce_conf"